project/frontend/src/components/RankingsList/RankingsListRow/RankingsListRow.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classes from "./RankingsListRow.module.css";
+
+const RankingsListRow = (props) => {
+    const getDiffIndicator = () => {
+        const diff =
+          props.competitor.rank < props.competitor.rankYesterday
+            ? "+"
+            : props.competitor.rank > props.competitor.rankYesterday
+            ? "-"
+            : "=";
+    
+        const style =
+          diff === "+"
+            ? { color: "green" }
+            : diff === "-"
+            ? { color: "red" }
+            : { color: "orange" };
```

Comment:
suggestion: if there's no change shouldn't the color just be black? otherwise a list of many competitors will be very colorful

Reply:
I will ask Mercury. It is currently consistent with the rest of the platform.

Reply:
I have it as orange for the other places for now - can change to black later if we decide it's too colorful but I find that black might stand out too much among lighter colours like red and green